loadflow/loadflow-validation/src/main/java/com/powsybl/loadflow/validation/ValidationConfig.java

```diff
@@ -218,12 +215,21 @@ public void setNoRequirementIfSetpointOutsidePowerBounds(boolean noRequirementIf
         this.noRequirementIfSetpointOutsidePowerBounds = noRequirementIfSetpointOutsidePowerBounds;
     }
 
+    /**
+     * Returns the loadflow configured or else the default platform loadflow.
+     */
+    public LoadFlow findLoadFlow() {
+        return getLoadFlowName()
+                .map(LoadFlow::find)
+                .orElseGet(LoadFlow::findDefault);
+    }
```

Comment:
Maybe the LoadFlow API should have a helper function to do this?

Reply:
Could you write a piece of code to show what could be added in `LoadFlow`?

Reply:
I'm not sure about the name of the method, but if we wrote this piece of code many times, we can imagine something to get a specified implementation and if the implementation is not found, return the default one.
```
LoadFlow.findPreferably("name");
```
I don't really know if it's useful or not...